### PR TITLE
Add Create Recovery Partition Installer.app v1.1

### PR DESCRIPTION
**Pull request by timsutton**
This is an app that makes it trivial to create an installer package that will install the recovery partition for a given version of OS X, using the same tools Apple does in their OS installer.

Thanks for your consideration!

### CODE REVIEW

**Comment by adityadalal924** [comment]
I noticed this hasn't been updated since 2013 - can you verify it works with El Capitan?

File: Casks/create-recovery-partition-installer.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'create-recovery-partition-installer' do
+  version '1.1'
+  sha256 '19dfb873cfa574bc0692e58b0c96554e8f59c8406bb720889da490a548400882'
+
+  url "https://github.com/MagerValp/Create-Recovery-Partition-Installer/releases/download/v#{version}/Create.Recovery.Partition.Installer-#{version}.dmg"
+  name 'Create Recovery Partition Installer'
```
Comment (vitorgalvao) [inline Casks/create-recovery-partition-installer.rb]: Please add a `appcast 'https://github.com/MagerValp/Create-Recovery-Partition-Installer/releases.atom'` line under.

File: Casks/create-recovery-partition-installer.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'create-recovery-partition-installer' do
+  version '1.1'
+  sha256 '19dfb873cfa574bc0692e58b0c96554e8f59c8406bb720889da490a548400882'
+
+  url "https://github.com/MagerValp/Create-Recovery-Partition-Installer/releases/download/v#{version}/Create.Recovery.Partition.Installer-#{version}.dmg"
```
Comment (vitorgalvao) [inline Casks/create-recovery-partition-installer.rb]: Please add a `appcast 'https://github.com/MagerValp/Create-Recovery-Partition-Installer/releases.atom'` line under.

**Comment by timsutton** [comment]
Yes, this tool does still work on the current versions of El Cap. I use it regularly on both 10.10 and 10.11 systems.

I noticed the appcast line in some other formulae but didn't see `appcast` listed in the documentation, and since this Cask still lists versions explicitly it wasn't really clear to me whether it was of any benefit. Anyway, it's added now. Thanks!

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in deffa58f6e2b4d07db606278a9b34a43829cccc6. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**